www/device.js

```diff
@@ -59,6 +60,7 @@ function Device () {
                 me.cordova = buildLabel;
                 me.model = info.model;
                 me.isVirtual = info.isVirtual;
+                me.isiOSAppOnMac = info.isiOSAppOnMac || false;
```

Comment:
```suggestion
                // isiOSAppOnMac is iOS specific. If defined, it will be appended.
                if (info.isiOSAppOnMac !== undefined) {
                    me.isiOSAppOnMac = info.isiOSAppOnMac;
                }
```